src/pages/term/Term.tsx

```diff
@@ -35,12 +49,31 @@ export default function Term(props: TermProps) {
 
     setTerm(term);
 
-    fetch(`/terms/${term.markdownPath}.markdown`)
+    fetch(`/terms/${term.customMarkdownPath}.md`)
```

Comment:
Add a case where `customMarkdownPath` is `undefined`.
```suggestion
    fetch(`/terms/${term.customMarkdownPath ?? term.pathUrl}.md`)
```